docs/source/ssl.rst

```diff
@@ -0,0 +1,23 @@
+.. _ssl:
+
+SSL
```

Comment:
I'd prefer to call this "TLS" everywhere. "SSL" is admittedly more well-known, but it's been a legacy name for 15 years...